blueoil/blueoil_convert.py

```diff
@@ -115,20 +119,20 @@ def run(experiment_id, restore_path, output_template_dir=None):
     """
 
     # Export model
-    export_dir = run_export(experiment_id, restore_path=restore_path)
+    export_dir = run_export(experiment_id, restore_path=restore_path, image_size=image_size)
 
     # Set arguments
     input_pb_path = os.path.join(export_dir, "minimal_graph_with_shape.pb")
-    dest_dir_path = export_dir
-    project_name = "project"
+    if project_name is None or len(project_name) < 1:
```

Comment:
Do you want to check empty string?

```
if not project_name:
```

is simple.

```
In [6]: not None                                                                                                                                                                                                                            
Out[6]: True

In [7]: not ""                                                                                                                                                                                                                              
Out[7]: True

In [8]: not "Hoge"                                                                                                                                                                                                                          
Out[8]: False
```

Reply:
THX!

Reply:
fixe! thank you for the comment :)